### PR TITLE
feat: group property on advanced settings

### PR DESCRIPTION
**Pull request by sshanzel**
Add the `group` property on `advancedSettings` type which is also part of the `FeedAdvancedSettings`.

WT-1985

### CODE REVIEW

**Review by kopancek — APPROVED** [review]

File: src/schema/feeds.ts [hunk 1]
```diff
@@ -67,11 +67,13 @@ export const typeDefs = /* GraphQL */ `
     title: String!
     description: String!
     defaultEnabledState: Boolean!
+    group: String!
   }
 
   type FeedAdvancedSettings {
     id: Int!
     enabled: Boolean!
+    advancedSettings: AdvancedSettings
```
[the remark this answers is not in the document]
Comment (omBratteng) [reply src/schema/feeds.ts:76]: Also probably need to update the GQL interfaces?

[the remark this answers is not in the document]
Comment (sshanzel) [reply src/schema/feeds.ts:76]: Yes, it is a property you can add in your query to pull the relevant `advancedSettings` when fetching this entity.

[the remark this answers is not in the document]
Comment (sshanzel) [reply src/schema/feeds.ts:76]: Ah, right. Totally missed the TypeScript interface.

**Review by idoshamun — COMMENTED** [review]
Can you add a test please for querying this field?

**Comment by sshanzel** [comment]
@idoshamun would this work? https://github.com/dailydotdev/daily-api/pull/1562/commits/207a85f1015b3d5b1964b447a803eac65c40f90d

**Pull request merged**